packages/@aws-cdk/aws-ses/test/test.receipt-rule-set.ts

```diff
@@ -53,6 +59,20 @@ export = {
       },
     }));
 
+    expect(stack).to(haveResource('AWS::SES::ReceiptRule', {
```

Comment:
Prefer creating a new test cases here, rather than piggy back on an existing test case.

Adding a test case like -

`test('drop spam rule should always appear first', ... ` makes it clear what the expectation is.

Reply:
Agreed.